pyteal/compiler/optimizer/optimizer_test.py

```diff
@@ -0,0 +1,417 @@
+from .optimizer import OptimizeOptions
+
+from pyteal.ir.ops import Op
+
+from pyteal import *
```

Comment:
nit: use relative import `.` instead of `pyteal`. Also, `Op` should already be included in the star import, no need to explicitly import it as well